_episodes/07-branches.md

```diff
@@ -80,13 +101,13 @@ $ git branch
 ~~~
 {: .output}
 
+Before we used the `checkout` command to checkout a commit using
```

Comment:
I like the addition of comparison to the original checkout.  I'm a bit worried it might be confusing saying we 'checked out the commit' because we checked out a file from a commit and stated this way it kind of references a detached head state which we avoided earlier.